### PR TITLE
Adds readme with setup and deploy commands

### PR DESCRIPTION
**Pull request by joaojusto**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#RubyConf Portugal '15 website
+
+The 2015 edition website, live [here](http://rubyconf.pt/)
+
+##Development
+
+To install the project depencies:
+
+```
+npm install
```
Comment (zamith) [inline readme.md]: This should be on a `bin/setup` script, but probably also on the `deploy` one.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+bundle install
+bower install
+```
+
+To start a development server with live reload:
+
+```
+grunt serve
```
Comment (zamith) [inline readme.md]: This should be `bin/server`

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+##Deploying
+
+To deploy to production simply run:
+
+```
+grunt deploy
```
Comment (zamith) [inline readme.md]: This should be `bin/deploy`

File: readme.md (hunk 1 above)
Comment (joaojusto) [inline readme.md]: Yes I'll create a bin/setup with this and also add to the bin/deploy

File: readme.md (hunk 2 above)
Comment (joaojusto) [inline readme.md]: Ok

File: readme.md (hunk 3 above)
Comment (joaojusto) [inline readme.md]: Ok

**Comment by joaojusto** [comment]
@zamith bump

**Comment by zamith** [comment]
:+1:

**Pull request merged**